docs/react_native_tutorials/react_native_1_setup_identifiers.md

```diff
@@ -0,0 +1,461 @@
+---
+id: react_native_1_setup_identifiers
+title: React Native Basic Setup & Create Identifiers
+sidebar_label: Setup & Create DIDs
+---
+
+This guide will walk you through setting up Veramo on React Native (using [expo](https://expo.dev/)). You should have a
+good understanding of React Native and have your environment set up correctly to build iOS and Android apps. Check out
+the [React Native](https://reactnative.dev/docs/environment-setup) docs to learn more. NodeJS v14 or later is required
+to run Veramo.
+
+You'll be creating a React Native application that is able to create and remember Decentralized Identifiers (DIDs) along
+with some keys associated with them and display them in a list.
+
+## Introduction
+
+Let's set up Veramo to run locally on the device and use `expo-sqlite` to store data, identities, and keys. Our
+identifier provider will be `did:ethr`. Initially, we will set up the [agent](../veramo_agent/introduction.md) in the
+most basic config and add more plugins for additional functionality as we go.
+
+Right now we just want to create an [identifier](../basics/identifiers.md).
+
+## Bootstrap React Native
+
+Use the [Expo CLI](https://docs.expo.dev/workflow/expo-cli/) bootstrap a new typescript project.
+As a prerequisite, please make sure to have `expo-cli` installed globally:
+
+```bash
+npm install -g expo-cli`
+```
+
+Then, we initialize a new project like so:
+
+```bash
+expo init VeramoMobile -t expo-template-blank-typescript
+cd VeramoMobile
+```
+
+Ensure your project is building and running ok before continuing to next steps (`npm run android` or `npm run ios`).
+
+## Install Dependencies
+
+### Prerequisite configuration
+
+In an ideal world we would install some dependencies, paste some sample code and start our app, but anyone who has
+worked with a non-trivial react-native app probably already knows that we don't currently live in this ideal world.
+Some libraries that we will be depending on make use of the node `crypto` package, or the `crypto.subtle` API of
+browsers. These are not yet available to react-native/expo environments out of the box, so there is a bit of ceremony to
+bundle everything properly.
+
+Also, the metro bundler that react-native uses doesn't yet support the `cjs` file extension (
+see [facebook/metro#535](https://github.com/facebook/metro/issues/535)) which is used by some libraries in the stack, so
+we have to configure these too.
+
+#### `cjs` extension
+
+Please adjust your `metro.config.js` file to look like this:
```

Comment:
when I got to this step in the tutorial, I did not have a `metro.config.js`. Where/when should this have been created?

Reply:
When I wrote this, a metro.config.js was created at the start.
If it's not, then it might work by just adding one?